### PR TITLE
Update INTERVIEWS.md

### PR DESCRIPTION
**Pull request by Goldies**

### CODE REVIEW

File: INTERVIEWS.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ We usually run more than one interview for each candidate.
 We also send them an introductory email presenting our company, our projects and our way of work.
 When running your interview, you should be aware if the interview is **technical** (run by developers) or **non-technical** (performed by our Tech Lead or Country Manager).
 
+Regardless of the type of interview, the intreview should last ONLY around 1 hour after which you should send a feedback email that is only a BRIEF summary with 1 or two paragraphs. In total between interview and feedback you should spend about 1.5 hours.
```
Comment (elbrujohalcon) [inline INTERVIEWS.md]: Can you change ALLCAPS by **bold**, please?

File: INTERVIEWS.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ We usually run more than one interview for each candidate.
 We also send them an introductory email presenting our company, our projects and our way of work.
 When running your interview, you should be aware if the interview is **technical** (run by developers) or **non-technical** (performed by our Tech Lead or Country Manager).
```
Comment (sheldonci) [inline INTERVIEWS.md]: I found 2 misspelled words in this sentence: 
- **non-technical**
- **technical**

File: INTERVIEWS.md (hunk 1 above)
Comment (sheldonci) [inline INTERVIEWS.md]: I found 1 misspelled word in this sentence: 
- intreview
	- *Suggested Replacements:* interview

File: INTERVIEWS.md [hunk 3]
```diff
@@ -7,6 +7,8 @@ We usually run more than one interview for each candidate.
...
 
+Regardless of the type of interview, the intreview should last ONLY around 1 hour after which you should send a feedback email that is only a BRIEF summary with 1 or two paragraphs. In total between interview and feedback you should spend about 1.5 hours. 
+
 #### Technical Interviews
```
Comment (sheldonci) [inline INTERVIEWS.md]: I found 1 misspelled word in this sentence: 
- ####

File: INTERVIEWS.md [hunk 4]
```diff
@@ -7,6 +7,8 @@ We usually run more than one interview for each candidate.
 We also send them an introductory email presenting our company, our projects and our way of work.
 When running your interview, you should be aware if the interview is **technical** (run by developers) or **non-technical** (performed by our Tech Lead or Country Manager).
 
+Regardless of the type of interview, the intreview should last **ONLY** around 1 hour after which you should send a feedback email that is only a **brief** summary with 1 or two paragraphs. In total between interview and feedback you should spend about **1.5 hours**.
```
Comment (sheldonci) [inline INTERVIEWS.md]: I found 5 misspelled words in this sentence: 
- hours**
- **1
- **brief**
- **ONLY**
- intreview
	- *Suggested Replacements:* interview

File: INTERVIEWS.md [hunk 5]
```diff
@@ -7,6 +7,8 @@ We usually run more than one interview for each candidate.
 We also send them an introductory email presenting our company, our projects and our way of work.
 When running your interview, you should be aware if the interview is **technical** (run by developers) or **non-technical** (performed by our Tech Lead or Country Manager).
 
+Regardless of the type of interview, the intreview should last **ONLY** around 1 hour after which you should send a feedback email that is only a **brief** summary with 1 or two paragraphs. In total between interview and feedback you should spend about **1.5 hours**. 
+
 #### Technical Interviews
```
Comment (sheldonci) [inline INTERVIEWS.md]: I found 1 misspelled word in this sentence: 
- ####

Comment (sheldonci) [inline INTERVIEWS.md]: I found 1 misspelled word in this sentence: 
- ####

File: INTERVIEWS.md (hunk 4 above)
Comment (sheldonci) [inline INTERVIEWS.md]: I found 5 misspelled words in this sentence: 
- hours**
- **1
- **brief**
- **ONLY**
- intreview
	- *Suggested Replacements:* interview

Comment (sheldonci) [inline INTERVIEWS.md]: I found 5 misspelled words in this sentence: 
- hours**
- **1
- **brief**
- **ONLY**
- intreview
	- *Suggested Replacements:* interview

File: INTERVIEWS.md (hunk 5 above)
Comment (sheldonci) [inline INTERVIEWS.md]: I found 1 misspelled word in this sentence: 
- ####

File: INTERVIEWS.md [hunk 6]
```diff
@@ -7,6 +7,8 @@ We usually run more than one interview for each candidate.
 We also send them an introductory email presenting our company, our projects and our way of work.
 When running your interview, you should be aware if the interview is **technical** (run by developers) or **non-technical** (performed by our Tech Lead or Country Manager).
 
+Regardless of the type of interview, the intreview should last **only** around 1 hour after which you should send a feedback email that is only a **brief** summary with 1 or two paragraphs. In total between interview and feedback you should spend about **1.5 hours**.
```
Comment (sheldonci) [inline INTERVIEWS.md]: I found 5 misspelled words in this sentence: 
- hours**
- **1
- **brief**
- **only**
- intreview
	- *Suggested Replacements:* interview

File: INTERVIEWS.md [hunk 7]
```diff
@@ -7,6 +7,8 @@ We usually run more than one interview for each candidate.
 We also send them an introductory email presenting our company, our projects and our way of work.
 When running your interview, you should be aware if the interview is **technical** (run by developers) or **non-technical** (performed by our Tech Lead or Country Manager).
 
+Regardless of the type of interview, the intreview should last **only** around 1 hour after which you should send a feedback email that is only a **brief** summary with 1 or two paragraphs. In total between interview and feedback you should spend about **1.5 hours**. 
+
 #### Technical Interviews
```
Comment (sheldonci) [inline INTERVIEWS.md]: I found 1 misspelled word in this sentence: 
- ####

**Pull request merged**